frontend/src/components/donation_page/index.jsx

```diff
@@ -32,8 +32,38 @@ class Donate extends Component {
             Wright, Founder/Executive Director at (773) 941-1200 or via email at{" "}
             <strong>Founder@TheYMIM.org</strong>
           </p>
+
+          <div className = "mx-auto">
+          <form
+            action="https://www.paypal.com/cgi-bin/webscr"
+            method="post"
+            target="_top"
+          >
+            <input type="hidden" name="cmd" value="_donations" />
```

Comment:
get rid of all the these hidden inputs and use these instead...

```
<input type="hidden" name="cmd" value="_s-xclick" />
<input type="hidden" name="hosted_button_id" value="34L62HLMP5VEC" />
```

something magical may happen 🐰